x/stakeibc/types/message_calibrate_delegation.go

```diff
@@ -0,0 +1,68 @@
+package types
+
+import (
+	"strings"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/Stride-Labs/stride/v14/utils"
+)
+
+const TypeMsgCalibrateDelegation = "calibrate_delegation"
+
+var _ sdk.Msg = &MsgCalibrateDelegation{}
+
+func NewMsgCalibrateDelegation(creator string, chainid string, valoper string) *MsgCalibrateDelegation {
+	return &MsgCalibrateDelegation{
+		Creator: creator,
+		ChainId: chainid,
+		Valoper: valoper,
+	}
+}
+
+func (msg *MsgCalibrateDelegation) Route() string {
+	return RouterKey
+}
+
+func (msg *MsgCalibrateDelegation) Type() string {
+	return TypeMsgCalibrateDelegation
+}
+
+func (msg *MsgCalibrateDelegation) GetSigners() []sdk.AccAddress {
+	creator, err := sdk.AccAddressFromBech32(msg.Creator)
+	if err != nil {
+		panic(err)
+	}
+	return []sdk.AccAddress{creator}
+}
+
+func (msg *MsgCalibrateDelegation) GetSignBytes() []byte {
+	bz := ModuleCdc.MustMarshalJSON(msg)
+	return sdk.MustSortJSON(bz)
+}
+
+func (msg *MsgCalibrateDelegation) ValidateBasic() error {
+	_, err := sdk.AccAddressFromBech32(msg.Creator)
+	if err != nil {
+		return errorsmod.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
+	}
+	if err := utils.ValidateAdminAddress(msg.Creator); err != nil {
+		return err
+	}
+
+	// basic checks on host denom
```

Comment:
comments seem off

Reply:
they were just stale from the copy paste - removed